A51-custom-backend-metrics.md

```diff
@@ -380,21 +415,122 @@ public interface OrcaOobReportListener {
 
 ### C++
 
-C-core's LB policies do have the ability to obtain backend metric data
-for both per-RPC and OOB mechanisms.  However, the C-core LB policy API
-is not yet ready to be made public, so this gRFC does not describe those
-APIs.  They will be described in a subsequent gRFC when the LB policy
-API is ready to be made public.
+#### Server
 
-This section describes only the C++ server-side APIs.
+C++ will provide `ServerMetricRecorder` to record metrics machine-wide.
+The user creates an instance and records metrics to it. All the methods are thread safe.
+This is optional for per-call load reporting and required for OOB load reporting
```

Comment:
Need a period at the end of this sentence.

Reply:
Done.